package.json

```diff
@@ -11,9 +11,11 @@
   },
   "devDependencies": {
     "@englercj/code-style": "^1.0.6",
-    "eslint": "^3.3.1"
-  },
-  "peerDependencies": {
+    "eslint": "^3.3.1",
     "jsdoc": "^3.4.0"
```

Comment:
jsdoc should remain a peer dependency, otherwise it will not notify users they need it when they install this module.